src/index.js

```diff
@@ -1,9 +1,11 @@
 import assign from 'object-assign';
 import pascalCase from 'pascal-case';
+import React from 'react';
```

Comment:
react is actually encouraging this because there should be one global react. @bradleyayers Would you prefer my first approach in https://github.com/webcomponents/react-integration/pull/33/commits/7f5ef29b99b049ba00bb1f246f9606464528af80?